src/main/scala/io/appthreat/atom/Atom.scala

```diff
@@ -227,6 +217,16 @@ object Atom:
             if x <= 0 then failure("`max-num-def` must be an integer larger than 0")
             else success
         )
+    arg[String]("input")
```

Comment:
Improved command line parsing when both full form and aliases are used. For example, before this change you cannot run atom with full arguments at the end as shown below.

Won't work.

```
atom -o ... --language java
```

Would work.

```
atom --language java -o ... 
```